N3.js/src/main/scala/org/w3/banana/n3js/N3.scala

```diff
@@ -0,0 +1,194 @@
+package org.w3.banana
+package n3js
+
+import scala.scalajs.js
+
+object N3 extends js.Object {
+
```

Comment:
"whatever" is a pretty dismissive remark, whereas this is perhaps the most important thing to understand about RDF. But ok, you can consider this ok, for this PR.